src/elements/ngm-cam-configuration.ts

```diff
@@ -23,6 +23,29 @@ import './ngm-cam-coordinates';
 
 type LockType = '' | 'elevation' | 'angle' | 'pitch' | 'move';
 
+/*
+ * Convert cartographic height (between -30'000m and +300'000) to input value (between 0 and 1)
+ * The input value between 0 to 0.5 is mapped to the height between -30'000m and 0m
+ * The input between 0.5 and 1 is mapped to the height between 0m and +300'000m
+ */
+function heightToValue(height: number): number {
+  const m = 0.5 / (height < 0 ? 30000 : 300000);
+  return m * height + 0.5;
+}
+
+/*
+ * Convert input value (between 0 and 1) to cartographic height (between -30'000m and +300'000)
+ * The input value between 0 to 0.5 is mapped to the height between -30'000m and 0m
+ * The input between 0.5 and 1 is mapped to the height between 0m and +300'000m
+ */
+function valueToHeight(value: number): number {
```

Comment:
What about creating unit tests for these 2 functions? Would be straightforward and a strong guarantee everything works as expected. Just export them and add a file to src/tests.